testfiles/LookupTables.idf

```diff
@@ -2671,7 +2671,7 @@
 
   Table:OneIndependentVariable,
     FanPowerRatioCurve,      !- Name
-    Exponent,                !- Curve Type
+    Cubic,                   !- Curve Type
```

Comment:
@mjwitte I'll put together a transition rule for this. If an "Exponent" curve type ever were used with "EvaluateCurveToLimits" it would have failed because there is no such regression defined. Another reason to drop the curve-fitting capability.